### PR TITLE
Killhttp and pingdom

### PR DESCRIPTION
**Pull request by dmorley**

### CODE REVIEW

File: db/add.php [hunk 1]
```diff
@@ -59,29 +59,14 @@
...
   $log->lwrite('Your pod has ssl and is valid ' . $_domain);
   echo 'Your pod has ssl and is valid<br>';
   $valid = true;
 }
-if (stristr($output, 'nodeName')) {
-  $log->lwrite('Your pod does not have ssl but is a valid pod ' . $_domain);
-  echo 'Your pod does not have ssl but is a valid pod<br>';
-  $valid = true;
-}
 if ($valid) {
```
Comment (noplanman) [inline db/add.php]: I think you can remove the `$valid` variable altogether and merge this block with the previous one on line 63:
`if (stristr($outputssl, 'nodeName')) {`

File: db/edit.php [hunk 2]
```diff
@@ -10,15 +10,15 @@
 $_weight     = $_GET['weight'] ?? '';
 $_email      = $_GET['email'] ?? '';
 $_oldemail   = $_GET['oldemail'] ?? '';
-$_statsurl   = $_GET['statsurl'] ?? '';
+$_stats_apikey   = $_GET['stats_apikey'] ?? '';
```
Comment (noplanman) [inline db/edit.php]: Align nicely to make easier to read 😇

File: showfull.php [hunk 3]
```diff
@@ -42,20 +42,11 @@
...
-      $class  = 'red';
-      $tip    = 'This pod does not offer SSL';
-    }
     $pod_name = htmlentities($row['name'], ENT_QUOTES);
-    $tip .= "\n This {$row['softwarename']} pod {$pod_name} has been watched for {$row['monthsmonitored']} months with an overall uptime of {$row['uptime_alltime']}% and a response time average today of {$row['responsetime']}ms was last checked on {$row['dateupdated']}. ";
+    $tip = "\n This {$row['softwarename']} pod {$pod_name} has been watched for {$row['monthsmonitored']} months with an overall uptime of {$row['uptime_alltime']}% and a response time average today of {$row['responsetime']}ms was last checked on {$row['dateupdated']}. ";
     $tip .= "On a scale of 100 this pod is a {$row['score']} right now";
 
-    echo '<tr><td><a title="' . $tip . '" data-toggle="tooltip" data-placement="bottom" class="' . $class . '" target="_self" href="/go.php?url=' . $scheme . $row['domain'] . '">' . $row['domain'] . '</a></td>';
+    echo '<tr><td><a title="' . $tip . '" data-toggle="tooltip" data-placement="bottom" class="' . $class . '" target="_self" href="/go.php?url=https://' . $row['domain'] . '">' . $row['domain'] . '</a></td>';
```
Comment (noplanman) [inline showfull.php]: `' . $class . '` should be `green`

File: show.php [hunk 4]
```diff
@@ -49,7 +42,7 @@
       $row['uptime_alltime'],
       $row['score']
     );
-    echo '<tr><td><div title="' . $tip . '" data-toggle="tooltip" data-placement="bottom"><a class="' . $class . ' url" target="_self" href="/go.php?url=' . $scheme . $row['domain'] . '">' . $row['domain'] . '</a></div></td>';
+    echo '<tr><td><div title="' . $tip . '" data-toggle="tooltip" data-placement="bottom"><a class="' . $class . ' url" target="_self" href="/go.php?url=https://' . $row['domain'] . '">' . $row['domain'] . '</a></div></td>';
```
Comment (noplanman) [inline show.php]: `$class` isn't defined and should be `text-success` from above.
Or better to use `green` like in `showfull.php` and `cleanup.php`.

Comment (dmorley) [inline show.php]: I am confused, on master its text-success for bootstrap. I have noticed a few things off from master, but develop is a branch of master, did I somehow make changes to master after you made a develop branch?

Comment (dmorley) [inline show.php]: Also on master I have titles on the lil social icons and they are not on develop, I think something mixed up

Comment (noplanman) [inline show.php]: If you compare with master, you'll see that you only added the titles in `show.php`.
This is a perfect example of code duplication and why it's difficult to maintain 😖 
Regarding the CSS class, same thing, it's different in different places. Best to search-replace and make them all the same.

Probably makes sense to rely on the CSS classes provided by bootstrap, and if they need a different styling, to just override them with custom properties.

**Comment by dmorley** [comment]
Think its pretty clean now

File: showfull.php [hunk 5]
```diff
@@ -92,10 +86,10 @@
...
     }
     echo '<td>';
-    $row['service_facebook'] === 't' && print '<div class="smlogo smlogo-facebook"></div>';
-    $row['service_twitter'] === 't' && print '<div class="smlogo smlogo-twitter"></div>';
-    $row['service_tumblr'] === 't' && print '<div class="smlogo smlogo-tumblr"></div>';
-    $row['service_wordpress'] === 't' && print '<div class="smlogo smlogo-wordpress"></div>';
+    $row['service_facebook'] === 't' && print '<div class="smlogo smlogo-facebook" title="Publish to Facebook" alt="Publish to Facebook"></div>';
+    $row['service_twitter'] === 't' && print '<div class="smlogo smlogo-twitter" title="Publish to Twitter" alt="Publish to Twitter"></div>';
+    $row['service_tumblr'] === 't' && print '<div class="smlogo smlogo-tumblr" title="Publish to Tumblr" alt="Publish to Tumblr"></div>';
+    $row['service_wordpress'] === 't' && print '<div class="smlogo smlogo-wordpress"  title="Publish to Wordpress" alt="Publish to Wordpress"></div>';
```
Comment (noplanman) [inline showfull.php]: As a WordPress aficionado I'm obliged to tell you that the only correct spelling of WordPress is **WordPress** 😇

File: db/pull.php [hunk 6]
```diff
@@ -291,7 +291,9 @@
       if ($uptr->monitors->monitor{'0'}->status == 9) {
         $status = 'Down';
       }
-      $pingdomdate = date('Y-m-d H:i:s');
+      if ($uptr) {
+      $statslastdate = date('Y-m-d H:i:s');
```
Comment (noplanman) [inline db/pull.php]: Alignment.

**Comment by noplanman** [comment]
99%, merge is imminent!

**Comment by dmorley** [comment]
Do you get notified when I push the changes up or do I have to comment?

**Comment by noplanman** [comment]
I get a notification :+1:

**Pull request merged**